_inc/client/search/index.jsx

```diff
@@ -156,6 +157,18 @@ export const SearchResults = ( {
 			}
 		}
 
+		if ( 'videopress' === element[0] ) {
+			if ( ! sitePlan || 'jetpack_free' === sitePlan.product_slug ) {
```

Comment:
videopress isn't on the personal plan either.

We *really* need a centralized system for this-- checking plans all over the place is a mess